web/admin/application/configs/klear/DdisList.yaml

```diff
@@ -34,6 +34,8 @@ production:
           externalCallFilter: true
           routeType: true
           target: true
+          ddiProvider: true
```

Comment:
ddiProvider?

Reply:
Yes, in order to add the description field last in the list, i have to add all previous fields in the order list.